MetaMask/metamask-mobile
# Remove condition to disable transaction confirm button if user has no balance

## **Description**
Currently we have these 2 conditions checked to disable submit button on confirmation pages:

1. account balance is less that gas + transaction value or
2. account balance is `0`

The second condition disables a `0` value transaction on a gasless network - something we support in MetaMask extension.

The PR removes the second condition and the first one alone is sufficient.

## **Manual testing steps**

1. Follow the steps listed here https://consensys.net/docs/goquorum/en/latest/tutorials/quorum-dev-quickstart/ to run Quorum locally
2. Run a DAPP locally and send transaction with value and gasPrice to `0`
3. Notice the "Submit" button is enabled


## **Related issues**
_Fixes #6567

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
